crates/cli/src/opts/build/zksync.rs

```diff
@@ -73,6 +102,11 @@ pub struct ZkSyncArgs {
 }
 
 impl ZkSyncArgs {
+    /// Returns true if zksync mode is enabled
+    pub fn enabled(&self) -> bool {
```

Comment:
Can rename this to `compile_enabled` or something to better relay the intent. If this is no longer significant we should remove this.

Reply:
Tbh if compile is true then zkVM usage is supported, otherwise if contracts aren't compiled for zksolc then we won't be able to switch to zkVM. 
Also, this is really only used for `create` where the config is loaded a bit weirdly so we retrieve this value from the cli args directly